### ISSUE TITLE
process.env.DATABASE_SERVICE_NAME is undefined

### ISSUE THREAD

**Issue by francoro** [sclorg/nodejs-ex#167]
I dont understand if i have to set the value for all the env variables : 
var mongoServiceName = process.env.DATABASE_SERVICE_NAME.toUpperCase(),
      mongoHost = process.env[mongoServiceName + '_SERVICE_HOST'],
      mongoPort = process.env[mongoServiceName + '_SERVICE_PORT'],
      mongoDatabase = process.env[mongoServiceName + '_DATABASE'],
      mongoPassword = process.env[mongoServiceName + '_PASSWORD']
      mongoUser = process.env[mongoServiceName + '_USER'];

And if i have to which value is for each one of these env variables.Please help.

**Comment by sfthurber** [comment]
no, you don't set these if you are deploying on openshift. see the working back end example at:
https://github.com/sfthurber/nodejs-ex
